godot-core/src/builtin/collections/array.rs

```diff
@@ -519,57 +521,98 @@ impl<T: ArrayElement> Array<T> {
         unsafe { duplicate.assume_type() }
     }
 
-    /// Returns a sub-range `begin..end`, as a new array.
+    /// Returns a sub-range `begin..end` as a new `Array`.
     ///
     /// The values of `begin` (inclusive) and `end` (exclusive) will be clamped to the array size.
     ///
+    /// If either `begin` or `end` are negative, their value is relative to the end of the array.
+    ///
+    /// # Example
+    /// ```no_run
+    /// # use godot_core::array;
```

Comment:
Try to use `godot::` imports, even in commented-out lines (here `godot::builtin::array`).